manifests/Python/Python/3.8.3.yaml

```diff
@@ -15,3 +15,4 @@ Installers:
       Switches:
         Silent: /Quiet
         SilentWithProgress: /Passive
+        Custom: PrependPath=1
```

Comment:
```suggestion
        Custom: PrependPath = 1
```
will it looked better?

Reply:
If you add spaces, python does not get added to the path.

Reply:
got it